docs/_tutorials/java-client.md

```diff
@@ -303,3 +303,30 @@ Client Username delete. Resp: 200
 At this point, you have created, retrieved, updated and deleted a Client Username object using SEMP. The examples used a generated client library in Java to interact with the Solace message router, but you can adapt the steps to any programming language of your choice. 
 
 SEMP is an extensive API that lets you configure anything on your Solace message router so there is a lot more to understand. If you want to know more, you can either get more familiar with the SEMP concepts by checking out the [Concepts Guide]({{ site.docs-concepts }}){:target="_top"} or you can see the full [developer documentation for the API]({{ site.docs-api }}){:target="_top"}.
+
+## Exploring the Samples
```

Comment:
This code seems better suited to a README.md in the java subdirectory. It's fairly generic.